src/openassetio-core/include/openassetio/Context.hpp

```diff
@@ -129,6 +129,16 @@ class Context final {
    */
   managerApi::ManagerStateBasePtr managerState{};
 
+  /**
+   * Constructs a new context.
+   *
+   * @warning This method should never be called directly by host code -
+   * @fqref{hostApi.Manager.createContext} "Manager.createContext"
+   * should always be used instead.
+   */
+  [[nodiscard]] static ContextPtr make(Access access = kUnknown, Retention retention = kTransient,
```

Comment:
[question] Is it ever valid to construct a `Context` with a `nullptr` `locale`?  Just wondering about the order of arguments here, i.e. we could put `locale` first without a default argument, to enforce that it is set.

Reply:
🤔 or maybe the default should be a new `TraitsData`?

Reply:
Added [this Issue](https://github.com/OpenAssetIO/OpenAssetIO/issues/499) to determine the most desirable behavior once we have the minimal vertical slice working.